gateway/src/apicast/policy/standalone/apicast-policy.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Do we need to expose this one?
I thought this one would be used transparently just when starting APIcast with `APICAST_CONFIGURATION`.

Reply:
Not needed 👍 It was generated by `bin/apicast generate policy`